modules/ROOT/pages/quickstart-server-installs.adoc

```diff
@@ -1,217 +1,97 @@
+----
 
-You can also check the status of the pods from the command line. Example:
+. Enter `ls -al` to view your Luna Streaming files:
 
-`kubectl get pods`
+[source,bash]
 ----
-NAME                                                 READY   STATUS      RESTARTS   AGE
-kotsadm-67c8d478f7-xs7mx                             1/1     Running     0          32m
-kotsadm-migrations                                   0/1     Completed   0          44m
-kotsadm-operator-75c844ffdc-k95q6                    1/1     Running     0          44m
-kotsadm-postgres-0                                   1/1     Running     0          44m
-kurl-proxy-kotsadm-79df797c77-jhvjb                  1/1     Running     0          43m
-prometheus-operator-984cbc6b8-g8ggz                  1/1     Running     0          8m45s
-prometheus-pulsar-single-kube-prometh-prometheus-0   2/2     Running     1          6m54s
-pulsar-autorecovery-5596f6464d-wtd22                 1/1     Running     2          8m45s
-pulsar-bastion-5567576b79-jrvjh                      1/1     Running     0          8m45s
-pulsar-bookkeeper-0                                  1/1     Running     0          8m45s
-pulsar-broker-77848bccb8-bj7bj                       1/1     Running     2          8m45s
-pulsar-dashboard-6cd77957b9-mqvd6                    2/2     Running     0          8m45s
-pulsar-function-0                                    2/2     Running     0          8m45s
-pulsar-proxy-59cc75485-88c5b                         3/3     Running     0          8m45s
-pulsar-pulsarmonitor-55c8c5454d-bq88d                1/1     Running     0          8m45s
-pulsar-single-grafana-78bfcd7848-n6jqh               2/2     Running     0          8m45s
-pulsar-single-kube-state-metrics-9c7644667-g9n27     1/1     Running     0          8m45s
-pulsar-single-prometheus-node-exporter-bk52h         1/1     Running     0          8m45s
-pulsar-zookeeper-0                                   1/1     Running     0          8m45s
-pulsar-zookeeper-metadata-r7r42                      0/1     Completed   0          8m45s
+➜  lunastreaming-2.10.0.3 ls -al
+total 88
+drwxr-xr-x@  11 firstname.lastname  staff    352 May 17 05:58 .
+drwx------+  98 firstname.lastname  staff   3136 May 24 14:15 ..
+-rw-r--r--@   1 firstname.lastname  staff  31209 Jan 22  2020 LICENSE
+-rw-r--r--@   1 firstname.lastname  staff   6612 Jan 22  2020 NOTICE
+-rw-r--r--@   1 firstname.lastname  staff   1269 Jan 22  2020 README
+drwxr-xr-x@  12 firstname.lastname  staff    384 Jan 22  2020 bin
+drwxr-xr-x@  21 firstname.lastname  staff    672 Jan 22  2020 conf
+drwxr-xr-x@   6 firstname.lastname  staff    192 May 17 05:58 examples
+drwxr-xr-x@   5 firstname.lastname  staff    160 May 17 05:58 instances
+drwxr-xr-x@ 277 firstname.lastname  staff   8864 May 17 05:58 lib
+drwxr-xr-x@  25 firstname.lastname  staff    800 Jan 22  2020 licenses
 ----
 
-== Viewing debug logs
+You have succesfully installed the DataStax Luna Streaming tarball. +
```

Comment:
Before this, add a step that point to OSS Pulsar bare metal installation document: https://pulsar.apache.org/docs/deploy-bare-metal/